### DISCUSSION TITLE
2006/10/06/%E6%98%A8%E5%A4%A9%E7%9C%8B%E4%BA%86%E3%80%8A%E5%A4%9C%E5%AE%B4%E3%80%8B

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#120] [Announcements]
# 2006/10/06/%E6%98%A8%E5%A4%A9%E7%9C%8B%E4%BA%86%E3%80%8A%E5%A4%9C%E5%AE%B4%E3%80%8B

我看之前，根本不知道故事情节。但网上对它的评论级差，不过我看了之后觉得还行，比评论的好。

https://blog.qizhen.xyz/2006/10/06/%E6%98%A8%E5%A4%A9%E7%9C%8B%E4%BA%86%E3%80%8A%E5%A4%9C%E5%AE%B4%E3%80%8B

**Comment by ruanqizhen** [comment]
之前博客下的留言：


Ling
2006年10月9日 09:29 编辑
我觉得最后不是悬念哦，而是想要表达说，对于权利地位的争夺永远没有止境。谁杀了皇后已经不重要了，重要的是皇后以为她得到了梦寐以求的王位，可实际上她却成为了另一个人夺取王位的牺牲品。个人意见个人意见：）
感觉比英雄，十面埋伏和无极还是要好看的，同学们咋就没点小马过河的精神涅。。。

回复
永青
2006年10月9日 09:23 编辑
厉帝的自杀是可以理解的，他的篡位也只是为了得到皇后，皇后一旦对他起了杀意，他也就觉得没意思了。
“有了嫂嫂，要江山干什么？”
这部片子真的不是一般的烂啊。。。

回复
任平生
2006年10月7日 00:27 编辑
这么一说，不用看了，这几部片子每一个好的……

回复
arrowpig
2006年10月6日 19:03 编辑
被你这么一说，一点看的兴趣也没有了。英雄，十面埋伏，无极，全是垃圾。有空还不如睡觉呢。

回复
